src/modules/handleToDo.js

```diff
@@ -0,0 +1,36 @@
+import setLocalStorage from './setLocalStorage.js';
+
+const todoContainer = document.querySelector('.dynamic-list');
+
+const handleToDo = (todo, todoItems) => {
+  const items = document.querySelectorAll('.todo');
+  items.forEach((item) => {
+    const idInNum = Number(item.querySelector('.section-1').getAttribute('data-time'));
+    if (idInNum === todo.ID) {
+      // mark complete
+
+      // delete todo
+      item.querySelector('.remove-btn').addEventListener('click', (e) => {
+        e.preventDefault();
+        todoContainer.removeChild(item);
+        const removeIndex = todoItems.indexOf(todo);
+        todoItems.splice(removeIndex, 1);
+        for (let i = 0; i < todoItems.length; i += 1) {
+          todoItems[i].ID = todoItems.indexOf(todoItems[i]) + 1;
+        }
+        setLocalStorage(todoItems);
+      });
+
+      // update todo
+      item.querySelector('.label').addEventListener('blur', () => {
+        const itemIndex = todoItems.indexOf(todo);
+        const currentItem = todoItems[itemIndex];
+        currentItem.Description = item.textContent;
```

Comment:
- [ ] Great job on the functionality but there is a little issue. When I edit the task description it is adding a new line for every character. Please fix this issue
 - To fix this you can use `item.innerText` instead of `item.textContent`

### See the image of the issue

![image](https://user-images.githubusercontent.com/83803414/161125539-fa1be317-8d75-4c3f-a836-ab67c3e228b0.png)